pymc3/variational/advi.py

```diff
@@ -138,7 +138,8 @@ def advi(vars=None, start=None, model=None, n=5000, accurate_elbo=False,
     # Create parameter update function used in the training loop
     uw_shared = theano.shared(uw, 'uw_shared')
     elbo = pm.CallableTensor(elbo)(uw_shared)
-    updates = optimizer(-1 * elbo, [uw_shared], learning_rate=learning_rate, epsilon=epsilon)
+    updates = optimizer([-1 * elbo], [uw_shared], learning_rate=learning_rate, epsilon=epsilon)
+    updates = apply_momentum(updates, [uw_shared], momentum=0.9)
```

Comment:
Always use momentum? I think we can disable mometum if you want stability in optimization.